test/local/account_routes.js

```diff
@@ -161,6 +165,104 @@ test(
 )
 
 test(
+  '/recovery_email/status with sign-in confirmation enabled',
+  function (t) {
+
+    var configOptions = {
+      signinConfirmation: {
+        enabled: true,
+        sample_rate: 1.0
+      }
+    }
+
+    var mockDB = {
+      deleteAccount: sinon.spy()
+    }
+
+    var mockRequest = {
+      auth: {
+        credentials: {
+          uid: uuid.v4('binary').toString('hex'),
+          email: TEST_EMAIL,
+          emailVerified: true,
+          tokenVerified: true
+        }
+      }
+    }
+
+    var accountRoutes = makeRoutes({
+      config: configOptions,
+      db: mockDB
+    })
+
+    var route = getRoute(accountRoutes, '/recovery_email/status')
+    return new P(function(resolve) {
+      route.handler(mockRequest, function(response) {
+        resolve(response)
+      })
+    })
+    .then(function(response) {
+      t.equal(mockDB.deleteAccount.callCount, 0)
```

Comment:
This can be safely removed, there is an explicit test case above, https://github.com/mozilla/fxa-auth-server/blob/edecb24ec75b6193bf2bda0ecdb2476801a5a427/test/local/account_routes.js#L95